packages/artifacts/chunks/languages/shapes.json

```diff
@@ -8,7 +8,7 @@
   ],
   "nodes": [
     {
-      "id": "/GC1oNIOzuK/OQrQEFJkrCWe88huyTaWVdfRy5Z1zFs=",
+      "id": "degCinkTmS4XeJW0Zt0gH5vpfm/A1DXx/661McUwJYE=",
```

Comment:
Are these ids unstable on purpose?

Reply:
Yes: I changed the name of the language, so the `hashIdGen` 'd IDs changed as well.